exonum-light-client/src/main/java/com/exonum/client/ExonumClient.java

```diff
@@ -17,9 +17,90 @@
 
 package com.exonum.client;
 
+import com.exonum.binding.common.hash.HashCode;
+import com.exonum.binding.common.message.TransactionMessage;
+import java.net.MalformedURLException;
+import java.net.URL;
+import okhttp3.OkHttpClient;
+
 /**
- * Exonum client main class to start with.
- */
+ * Main class to start using Exonum Light client.
+ * Provides a convenient way for interaction with Exonum framework APIs.
+ * It uses {@linkplain OkHttpClient} internally for REST API communication with Exonum node.
+ **/
 public interface ExonumClient {
-  //TODO: implement
+
+  /**
+   * Submits the transaction message to an Exonum node.
+   * @return transaction message hash
+   * @throws RuntimeException if network request fails
```

Comment:
What if we use a more general statement for the interface:
```
if the client is unable to submit the transaction (e.g., in case of connectivity problems)
```